sdk/identity/identity/samples/javascript/README.md

```diff
@@ -0,0 +1,67 @@
+---
+page_type: sample
+languages:
+  - javascript
+products:
+  - azure
+  - azure-identity
```

Comment:
`azure-identity` isn't a valid product slug. Unfortunately this is one of those things we have to check manually, as there's an exhaustive list of which product strings are valid here. We should probably use `azure-active-directory` instead.